.github/workflows/rust-workflow.yml

```
@@ -26,3 +26,13 @@ jobs:
       clippy_args: '--all-features'
       test_args: '--all-features'
       testcov_args: '--all-features'
+
+  publish:
+    needs: check
+    uses: famedly/backend-build-workflows/.github/workflows/publish-crate.yml@fd0845e44d45187e62d65e6af5b0193ccd655feb
+    with:
+      registry-name: "crates-io"
+      registry-index: "https://github.com/rust-lang/crates.io-index"
+    secrets:
+      CI_SSH_PRIVATE_KEY: ${{ secrets.CI_SSH_PRIVATE_KEY }}
+      registry-auth-token: ${{ secrets.CRATES_IO_TOKEN }}
```

I think you can just do

```suggestion
    secrets: inherit
```

---

The secrets don't contain a `registry-auth-token`, so it won't magically remap that, right?

---

ah right nevermind